app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h2>All the post of a user goes here</h2>
\ No newline at end of file
+<div class="posts-container">
+    <div class="post-header">
+        <div class="post-image-holder">
+            <img src="<%= @user.photo %>" alt="user">
+        </div>
+            <div class="post-details">
+                <h3><%= @user.name %></h3>
+                <p>Posts: <%= @user.posts_counter || 0%></p>
+        </div>
+    </div>
+
+    <% @posts.reverse.each_with_index do |post, index| %>
+        <div class="post">
+            <a href="/users/id/posts/<%= post.id %>"><h4>Post No: <%= index+1 %></h4></a>
+                
+            <h3><%= post.title %></h3>
```

Comment:
- [ ] kindly uses `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page

Reply:
post.recent_comments already used in posts/index.html.erb file

Reply:
@andersonlebon